ZLPeoplePickerViewController/ZLPeoplePickerViewController.m

```diff
@@ -237,9 +256,41 @@ - (void)tableView:(UITableView *)tableView
 
     if ([self.selectedPeople containsObject:contact.recordID]) {
         [self.selectedPeople removeObject:contact.recordID];
+        UIButton *cancelButton;
+        UIView *topView = _searchController.searchBar.subviews[0];
+        for (UIView *subView in topView.subviews) {
+            if ([subView isKindOfClass:NSClassFromString(@"UINavigationButton")]) {
+                cancelButton = (UIButton*)subView;
+            }
+        }
+        if (cancelButton) {
+            if ([self.selectedPeople count] != 0) {
+                [cancelButton setTitle:@"Done" forState:UIControlStateNormal];
+            }
+            else{
+                [cancelButton setTitle:@"Cancel" forState:UIControlStateNormal];
+            }
+            
+        }
     } else {
         if (self.selectedPeople.count < self.numberOfSelectedPeople) {
             [self.selectedPeople addObject:contact.recordID];
+            UIButton *cancelButton;
+            UIView *topView = _searchController.searchBar.subviews[0];
+            for (UIView *subView in topView.subviews) {
+                if ([subView isKindOfClass:NSClassFromString(@"UINavigationButton")]) {
+                    cancelButton = (UIButton*)subView;
+                }
+            }
+            if (cancelButton) {
```

Comment:
You could make the search for `cancelButton` as a category of UISearchBar. Because you're using the same code here and above. I would advise to use a method name such as `zl_cancelButton` because it may still cause naming conflicts for Objective-C codebase.